### DISCUSSION TITLE
We can convert the page to use Nextjs and typescript

### DISCUSSION THREAD

**Discussion by gabriel-logan** [LyubomirT/lookaround#10] [Ideas]
The project becomes more professional, organized, better and safer, in addition to being easier to maintain and scale.

**Comment by LyubomirT** [comment]
Certainly possible, but could you please elaborate a bit on this? I agree with the first two points, but the last ones seem to be too general.

**Reply by gabriel-logan** [reply]
Then I can make a new pr to help with the use of the page, on the cell phone the design is a little strange

**Reply by LyubomirT** [reply]
That'd be much appreciated! I believe this is a great improvement for the website, thank you for considering to contribute to this project again. :)

**Reply by LyubomirT** [reply]
There was already a PR that tried to implement that (#8), but it appears to be abandoned after a misunderstanding between me and the author. You can get started from scratch or try to use any code from their fork if you want.

**Comment by gabriel-logan** [comment]
If you know nextjs, it's a great idea to use it, because it's ssr, so you can make a full stack application directly with it.  The other point is that it becomes more scalable, for now the project is small and there is no problem.  I say more because they want to scale the project to something better, so nextjs would make a lot of difference.

Typescript too, for better security

**Comment by LyubomirT** [comment]
True, although this project is more focused on front-end as most manipulations happen there (via Three.js), back-end is only there for the sake of storing the image temporarily, and it's not like any sensitive data is uploaded. Perhaps for a true full-stack app that will be acceptable, but as of now it's not really needed and can be considered overkill.
